components/eamxx/src/physics/mam/eamxx_mam_aci_process_interface.hpp

```diff
@@ -20,7 +20,15 @@ class MAMAci final : public scream::AtmosphereProcess {
 
   // views for single- and multi-column data
   using const_view_2d = typename KT::template view_2d<const Real>;
-  
+  using view_2d       = typename KT::template view_2d<Real>;
+  // rho is air density [kg/m3]
+  view_2d rho_;
+
+  // w0_ is large scale velocity (m/s)
+  view_2d w0_;
+
+  // Top level for troposphere cloud physics
+  const int top_lev_ = 6;
```

Comment:
Maybe add a "MUST FIXME" comment here so that we remove this hardwired number latter.